tests/functional/features/chart_src_without_report.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Chart source only submission
+    Partners or redhat associates can publish their chart by submitting
+    error-free chart in source format without the report.
+
+    Scenario: The partner hashicorp submits a error-free chart source for vault
+        Given hashicorp is a valid partner
+        And hashicorp has created an error-free chart source for vault
+        When hashicorp sends a pull request with the vault source chart
+        Then hashicorp should see the pull request getting merged
+        And the index.yaml file is updated with a new entry
```

Comment:
Maybe "the index.yaml file is updated with an entry for the submitted chart"

Reply:
:+1: